FLEDGE_Key_Value_Server_API.md

```diff
@@ -65,6 +65,12 @@ HTTPS is used to transport data. The method is `POST`.
 
 The HTTP POST body is encrypted.
 
+#### Cleartext headers
+
+Requests contain a cleartext HTTP `Content-Type` header with value `message/ad-auction-trusted-signals-request`.
+
+Responses contain a cleartext HTTP `Content-Type` header with value `message/ad-auction-trusted-signals-response`.
```

Comment:
I don't think we can necessarily specify what a response does contain, I think we can specify what it must contain.  Further, this begs the question of who adds this response header?  the TKV code or the HTTPS-terminating front-end in front of the TKV?  I think the HTTPS-terminating front-end.